scripts/bench/README.md

```diff
@@ -1,40 +1,30 @@
-Work-in-progress benchmarks.
+# React Benchmarking
 
-## Running the suite
+## Commands
 
-You'll need two folders to compare, each of them containing `react.min.js` and `react-dom-server.min.js`. You can run `npm run build` at the repo root to get a `build` folder with these files.
+```bash
+# will compare local repo vs remote merge base repo
+yarn bench
 
-For example, if you want to compare a stable verion against master, you can create folders called `build-stable` and `build-master` and use the benchmark scripts like this:
+# will comapre local repo vs remote merge base repo
+# this can significantly improve bench times due to no build
+yarn bench -- --skip-build
 
-```
-$ ./measure.py build-stable stable.txt build-master master.txt
-$ ./analyze.py stable.txt master.txt
-```
+# will only build and run local repo against benchmarks
+yarn bench -- --local
 
-The test measurements (second argument to `analyze`, `master.txt` in this example) will be compared to the control measurements (first argument to `analyze`, `stable.txt` in this example).
+# will only build and run remote merge base repo against benchmarks
+yarn bench -- --remote
 
-Changes with the `-` sign in the output mean `master` is faster than `stable`.
+# will only build and run remote master repo against benchmarks
+yarn bench -- --remote=master
 
-You can name folders any way you like, this was just an example.
+# same as "yarn build"
+yarn bench -- --remote --local
 
-## Running one
-One thing you can do with them is benchmark initial render time for a realistic hierarchy:
+# runs benchmarks with Chrome in headless mode
+yarn bench -- --headless
```

Comment:
I've had no official say-so for perf testing. I'll chase someone to get some feedback on using it before enabling it by default.